README.md

```diff
@@ -1,17 +1,139 @@
-# Registers Ruby Client
+# GOV.UK Registers Ruby Client
+
+This documentation is for developers interested in using this Ruby client to integrate their service with GOV.UK Registers. 
+
 ## Installation
 In your Gemfile add:
 ```
 gem 'registers-ruby-client', git: 'https://github.com/openregister/registers-ruby-client.git'
 ```
 ## Usage
+
+_Note: examples use the `country` register_
+
 ```
 require 'registers_client'
 
 registers_client = RegistersClient::RegistersClientManager.new({ cache_duration: 3600 })
+
+
 register_data = registers_client.get_register 'country', 'beta'
-register_data.get_records.first[:item]
- => {"citizen-names"=>"Soviet citizen", "country"=>"SU", "end-date"=>"1991-12-25", "name"=>"USSR", "official-name"=>"Union of Soviet Socialist Republics"} 
- ```
+```
+
+### Methods
+
+#### `get_entries`
+
```

Comment:
I think we should describe each of these methods in a human readable sentence. We can re-use stuff from the API docs where the methods are equivalent.

Reply:
Have added human-readable descriptions in f4eaa04ac550a00a8020d61b7f61baef2158aefd

Please review these. I'm not sure about `get_metadata_records`: could you help me out with that one?

Reply:
Erm, looking at this method it is a bit superfluous, as everything it returns is available through other methods. However, if you want to document it as-is you could say something like.
*All metadata records of the register, this includes the register definition, field definitions and custodian.*

Reply:
I thought we should document it if it's exposed to the user, but if you think we can leave it out then I will follow your lead. 

Does it definitely include all of that, though? With the `country` register it only seems to return the register name for me.

Reply:
@bahmady yes, that is just related to the query you are using if you do:
```
register_data.get_metadata_records.each do |m|
  puts m.to_json
end
```
you can see all the records.